src/arch/armv8/armv8-r/aarch64/boot.S

```diff
@@ -129,10 +129,8 @@ skip_non_loadable:
     isb
 
     /* Enable caches and MPU */
-    mrs x3, sctlr_el2
-    ldr x4, =(SCTLR_C | SCTLR_I | SCTLR_M)
-    orr x3, x3, x4
-    msr sctlr_el2, x3
+    ldr x4, =(SCTLR_RES1_AARCH32 | SCTLR_C | SCTLR_I | SCTLR_M)
```

Comment:
You are using 'SCTLR_RES1_AARCH32' in aarch64 boot. Please fix with 'SCTLR_RES1'

Reply:
You're right, thanks!